testing_and_setup/compass/ocean/soma/soma_run.xml

```diff
@@ -4,10 +4,6 @@
 		<option name="config_ocean_run_mode">'forward'</option>
 		<option name="config_use_mom_del4">.true.</option>
 		<option name="config_mom_del4_div_factor">10.0</option>
-		<option name="config_convective_visc">1.0</option>
-		<option name="config_convective_diff">1.0</option>
-		<option name="config_use_rich_visc">.true.</option>
-		<option name="config_use_rich_diff">.true.</option>
```

Comment:
No addition to compensate for SOMA?

Reply:
This looks like it was a really odd configuration with CVMix running with KPP and yes also with Richardson vertical mixing also turned on.  That makes no sense and isn't something we can duplicate with CVMix on its own.

Reply:
CVmix should have been turned off, which is a bug then.  From the SOMA paper (Wolfram et al 2015) the specification is (noting the committed code was done post-facto because SOMA predates COMPASS):

We use the Richardson number–based parameterization of Pacanowski and Philander (1981) as our ver- tical mixing model. Background values of nu_v = 106-4 m^2/s and kappa_v 10^-5 m^2/s are used for viscosity and diffusivity away from the mixed layer and convective mixing. The coefficient for Richardson number vertical mixing function is that suggested by Pacanowski and Philander (1981), namely, C_Ri = 0.005 m^2/s.  Vertical mixing values of nu_v=kappa_v=1 m^2/s are used to remove gravitationally unstable profiles.

Reply:
Good to know.  Could you make suggested edits to the configuration, then?

Reply:
The case is in production by @lizcarlson -- is this useful @lizcarlson?  Don't want to change anything out from underneath your work, but also don't want you to use a case that isn't as promised in the paper (because of the basins to COMPASS transition).

Reply:
That file had both cvmix and the old rich mixing on. I removed the old rich flags, but these remain:
```
  7         <option name="config_use_cvmix">.true.</option>
  8         <option name="config_use_cvmix_background">.true.</option>
  9         <option name="config_use_cvmix_convection">.true.</option>
 10         <option name="config_cvmix_background_diffusion">5.0e-6</option>
 11         <option name="config_cvmix_convective_basedOnBVF">.true.</option>
 12         <option name="config_use_cvmix_shear">.true.</option>
 13         <option name="config_cvmix_shear_mixing_scheme">'KPP'</option>
 14         <option name="config_use_cvmix_kpp">.true.</option>
```
They were added in 2d229ed6db (by Wolfram 2016-12-07)

Based on your paragraph above, I think you want:
```
config_use_cvmix = .true.     (this is now default true)
config_use_cvmix_background = .true.
config_cvmix_background_diffusion = 1.0e-5
config_cvmix_background_viscosity = 1.0e-4
config_use_cvmix_convection = .true.

config_use_cvmix_shear = .true.
config_cvmix_num_ri_smooth_loops = 2
config_cvmix_use_BLD_smoothing = .true.
config_cvmix_shear_mixing_scheme = 'PP'
config_cvmix_shear_PP_nu_zero = 0.005
```
If you agree, I can update the soma file. The main change is from KPP to PP. That looks like a mistake in the original.

Reply:
@mark-petersen the specification you say here is consistent with what @pwolfram writes above.  It turns on the default PP behavior.  This shouldn't impact @lizcarlson whose simulations are complete.  I've verified with her mixing was configured appropriately

Reply:
Awesome, so very glad to hear!!!  Thanks @vanroekel